Dockerfile

```
@@ -11,7 +11,7 @@ RUN npm run build:prod
 
 
 # Build Python app - this stage is a common base for the prod and dev stages
-FROM python:3.11-bullseye AS backend
+FROM python:3.12-bullseye AS backend
```

```suggestion
FROM python:3.12-bookworm AS backend
```

ref: https://endoflife.date/debian

---

Good one Dan!